src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,163 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> node;
```

Comment:
According to the checklist, class `Node<T>` should be an inner class. It's better to remove the `node` field from `MyLinkedList` class and create instances of `Node<T>` where it's necessary.